rivescript.go

```diff
@@ -87,7 +90,16 @@ func New(cfg *Config) *RiveScript {
 		cfg.Depth = 50
 	}
 	if cfg.SessionManager == nil {
-		cfg.SessionManager = memory.New()
+		if cfg.Persist == true && cfg.DBfile != "" {
+			var err error
+			cfg.SessionManager, err = skv.New(cfg.DBfile)
+			if err != nil {
+				fmt.Printf("Error Opening Database Try turning persist off.\n", err)
+				os.Exit(1)
+			}
+		} else {
+			cfg.SessionManager = memory.New()
+		}
```

Comment:
I'd prefer that third-party session managers _not_ be referenced directly in the core of rivescript-go. Users who want to use the default in-memory store or the Redis one (for example) shouldn't need to install the Go dependencies for rapidloop/skv if they don't want to use this one.

Usage of this session store should instead be up to the author of the bot program:

```go
package main

import (
    "github.com/rapidloop/skv"
    "github.com/aichaos/rivescript-go"
)

func main() {
    skv, _ := skv.New("my.db")
    bot := rivescript.New(&rivescript.Config{
        SessionManager: skv,
    })
}
```

See the [redis driver](https://github.com/aichaos/rivescript-go/tree/master/sessions/redis) for example.